parts/windows/kuberneteswindowsfunctions.ps1

```diff
@@ -214,6 +214,16 @@ function Register-NodeResetScriptTask {
     Register-ScheduledTask -TaskName "k8s-restart-job" -InputObject $definition
 }
 
+function Register-NodeLabelSyncScriptTask {
+    Write-Log "Creating a periodical task to run windowsnodelabelsync.ps1"
+
+    $action = New-ScheduledTaskAction -Execute "powershell.exe" -Argument "-File `"c:\k\windowsnodelabelsync.ps1`""
+    $principal = New-ScheduledTaskPrincipal -UserId SYSTEM -LogonType ServiceAccount -RunLevel Highest
+    $trigger = New-JobTrigger -AtStartup -RandomDelay 00:01:00  -RepetitionInterval 00:01:00

```

Comment:
1. What is the frequency in Linux? 1 minute may be too frequent.
2. Should we delay 1 minute since the interval is 1 minute?

Reply:
> What is the frequency in Linux? 1 minute may be too frequent.

It's 1 minute, could you please take a look at my PR description, it will help the reviewer get the background of this change.

> Should we delay 1 minute since the interval is 1 minute?

Do you mean a fixed delay?

Reply:
For 1, I did not find 1 minute for Linux in PR's description.
For 2, is delay needed? Can we remove delay?

Reply:
We can discuss.
In linux implementation, the systemd service, e.g. node-labe.service,  use `RestartSec=60` to sleep 60s  before restarting the service after exiting.

https://github.com/Azure/AgentBaker/pull/471/files#diff-1d6a169665921e5390c155a703d65896550b257e021e945c8824f12fed40b680R5

Reply:
I think that we may need some test data for this setting. 1 minute may be too aggressive especially when user's workload is heavy.